cli/test/commands/transfer/index.test.ts

```diff
@@ -1,19 +1,24 @@
 import { expect, test } from "@oclif/test";
 import { initTestEnvIfNeeded } from "../../../src/utils/initTestEnv";
-import { getPayer, getSolanaRpcUrl } from "../../../src";
-import { Connection, PublicKey } from "@solana/web3.js";
+import {
+  defaultSolanaWalletKeypair,
+  getPayer,
+  getSolanaRpcUrl,
+} from "../../../src";
+import { Connection, Keypair, PublicKey } from "@solana/web3.js";
 import { createMint, mintTo } from "@lightprotocol/compressed-token";
 import { requestAirdrop } from "../../helpers/helpers";
 describe("transfer", () => {
   test.it(async () => {
     await initTestEnvIfNeeded();
-    await requestAirdrop(getPayer().publicKey);
-
+    const mintKeypair = defaultSolanaWalletKeypair() || Keypair.generate();
+    await requestAirdrop(mintKeypair.publicKey);
     const mintAmount = 10;
-    const mintAuthority = getPayer().publicKey;
-    const mintDestination = getPayer().publicKey;
-    const mintAddress = await createTestMint();
+    const mintAuthority = mintKeypair.publicKey;
+    const mintDestination = mintKeypair.publicKey;
+    const mintAddress = await createTestMint(mintKeypair);
```

Comment:
I think the mintAuthority and mintKeypair should be different keypairs so that it is realistic.
(The mint keypair will become a Solana account. You don't want to turn your wallet keypair into a mint account.)